components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -90,14 +90,31 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Computed>("LW_flux_up", scalar3d_layout_int, Wm2, grid->name(), ps);
   add_field<Computed>("LW_flux_dn", scalar3d_layout_int, Wm2, grid->name(), ps);
 
+  // Translation of variables from EAM
+  // --------------------------------------------------------------
+  // EAM name | EAMXX name       | Description
+  // --------------------------------------------------------------
+  // soll       sfc_flux_dir_nir   solar near-IR direct flux
+  // sols       sfc_flux_dir_vis   solar UV/visible direct flux
+  // solld      sfc_flux_dif_nir   solar near-ID diffuse flux
+  // solsd      sfc_flux_dif_vis   solar UV/visible diffuse flux
+  // --------------------------------------------------------------
```

Comment:
Should we add a description for `sfc_flux_sw_net` and `sfc_flux_lw_dn`? If this PR passes test and we don't want to delay getting it in, I can add it to the SC PR that follows this. 

```
// EAM name | EAMXX name       | Description
// netsw           sfc_flux_sw_net             ?
// flwds            sfc_flux_lw_dn               ?
```

Reply:
Oh yeah, I had it in here but took it out because I moved where I set these to the kokkos parallel for block below. I'll add a comment back in.

Reply:
Okay done.